schedule/kernel/install_ltp_uefi_baremetal.yaml

```diff
@@ -3,16 +3,32 @@ description:    >
     basic installation testing on baremetal UEFI + install LTP
 vars:
     AUTOYAST_PREPARE_PROFILE: 1
```

Comment:
Do we need this variable even after change?

Reply:
of course not, thanks for spotting this!